filament/src/details/Renderer.cpp

```diff
@@ -298,7 +298,10 @@ bool FRenderer::beginFrame(FSwapChain* swapChain, uint64_t vsyncSteadyClockTimeN
         FEngine& engine = mEngine;
         FEngine::DriverApi& driver = engine.getDriverApi();
 
-        driver.beginFrame(appVsync.time_since_epoch().count(), mFrameId);
+        driver.beginFrame(
+                appVsync.time_since_epoch().count(),
+                int64_t(1'000'000'000.0 / mDisplayInfo.refreshRate),
```

Comment:
[comment] mentions it that `refreshRate` is `0` for offscreen, is the comment outdated? 

[comment]: https://github.com/google/filament/blob/main/filament/include/filament/Renderer.h#L77

Reply:
ah good catch. I should probably pass 0 in that case.